source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -304,6 +310,11 @@ export default class NetworkStatusStore extends Store {
     return Promise.resolve();
   };
 
+  _handleLogStateSnapshot = (): Promise<void> => {
```

Comment:
@DeeJayElly please add a comment above this line with an explanation of what this method does 🙏

Reply:
Done